apps/rod/sections/summary-data-sections.js

```diff
@@ -20,46 +20,59 @@ module.exports = {
       step: '/sponsor-type',
       field: 'sponsor-type'
     },
+    {
+      step: '/application',
+      field: 'application-type'
+    },
+    {
+      step: '/visa-type',
+      field: 'visa-type'
+    },
     {
       step: '/about-application',
       field: 'date-of-application',
       parse: d => d && dateFormater.format(new Date(d))
     },
+
     {
       step: '/about-application',
       field: 'cancel-application'
     },
     {
-      step: '/dependant-or-guardian',
-      field: 'dependant-or-guardian'
-    },
-    {
-      step: '/legal-representation',
-      field: 'legal-rep-name'
+      step: '/main-applicant-passport',
+      field: 'is-requesting-passport-to-travel'
     },
     {
-      step: '/application',
-      field: 'application-type'
+      step: '/further-leave',
+      field: 'further-leave-to-remain'
     },
     {
-      step: '/visa-type',
-      field: 'visa-type'
+      step: '/reference-number',
+      field: 'rod-reference-number',
+      parse: value => {
+        return Array.isArray(value) ? value.map(option => option).join('\n') : value;
+      }
     },
     {
-      step: '/further-leave',
-      field: 'further-leave-to-remain'
+      step: '/reference-number',
+      field: 'rod-case-id'
     },
     {
-      step: '/about-application',
-      field: 'cancel-application'
+      step: '/reference-number',
+      field: 'rod-ho-reference-number'
     },
```

Comment:
we need to show payment reference and courier reference as well. Figma might be showing only two but that depends on how many reference items are selected.